mvrx/src/main/kotlin/com/airbnb/mvrx/BaseMvRxViewModel.kt

```diff
@@ -51,23 +83,39 @@ abstract class BaseMvRxViewModel<S : MvRxState>(
     private val lifecycleOwner: LifecycleOwner = LifecycleOwner { lifecycleRegistry }
     private val lifecycleRegistry: LifecycleRegistry = LifecycleRegistry(lifecycleOwner).apply { currentState = Lifecycle.State.RESUMED }
 
+    /**
+     * Synchronous access to state is not exposed externally because there is no guarantee that
+     * all setState reducers have run yet.
+     */
     internal val state: S
         get() = stateStore.state
 
+    /**
+     * Return the current state as a Flow. For certain situations, this may be more convenient
+     * than subscribe and selectSubscribe because it can easily be composed with other
+     * coroutines operations and chained with operators.
+     */
+    val stateFlow: Flow<S>
+        get() = stateStore.flow
+
+    @VisibleForTesting
+    internal val bufferedStateFlow: Flow<S>
+        get() = stateStore.flow.buffer()
```

Comment:
actually, you don't need to buffer state flow after switching from StateFlow to BroadcastChannel, because you're already have buffer in `CoroutineStateStore` and producer supports backpressure correctly (in [this ](https://github.com/airbnb/MvRx/pull/385/commits/acf024a2c414436fbb131df2e39272b02bd17c26)commit in another PR). 

Buffering state here is actually just increases buffer size twice from 64 to 128 - 64 per subscriber, and 64 common buffer. 
Also, as one more side effect, a 64-element buffer will be created for each subscriber which may not be memory efficient

Reply:
@denis-bezrukov I was thinking that the first buffer would be for fast sets and this one could account for slow consumers. Wouldn't they serve two different purposes?
Without this, what would happen to other consumers if there was one slow one?

Reply:
@denis-bezrukov. I was able to remove this (but kept channelFlow above) and all tests still passed so I think I am okay removing this and saving the memory.

Reply:
@gpeal ArrayBroadcastChannel's buffer behaves the same way - slow consumer won't prevent fast from collecting everything in the buffer. Slow consumer only prevents sending buffer+1 element to it. So adding one more buffer here is just increases the total buffer size, but less effective than making it 128 at BroadcastChannel. 

Sample: total buffer size 10, send 20 items to it:
Channel's buffer only: https://pl.kotl.in/SoU1QXYxH
Channel's buffer + flow buffer (5 + 5): https://pl.kotl.in/8YWk_V25V

In both cases fast consumer quickly consumes first 10 items (total buffer size) and only then it slows down by the slow consumer the same way in both cases.